### PR TITLE
Closes #75 Add views index and show for admin

### LINKED ISSUE #75: Admin views an individual Order
As an authenticated Admin, when I visit an individual order page
Then I can see the order's date and time.
And I can see the purchaser's full name and address.
And I can see, for each item on the order:
    - The item's name, which is linked to the item page.
    - Quantity in this order.
    - Price
    - Line item subtotal.
And I can see the total for the order.
And I can see the status for the order.

### PR DESCRIPTION
**Pull request by Tman22**
Added views for index and show page. I added the shipping information about  the user in the show order partial.

### CODE REVIEW

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Admin::OrdersController < Admin::BaseController
```
Comment (Tman22) [inline app/controllers/admin/orders_controller.rb]: I believe we decided to ignore this.

File: test/integration/admin_views_all_orders_test.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require "test_helper"
+
+class AdminViewsAllOrdersTest < ActionDispatch::IntegrationTest
```
Comment (ToniRib) [inline test/integration/admin_views_all_orders_test.rb]: Can we change the filename & the test name to "Admin Views An Individual Order" so it matches the title/card for issue #75?

File: test/integration/user_can_view_items_from_an_individual_artist_test.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+require "test_helper"
+
+class UserCanViewItemsFromAnIndividualArtistTest < ActionDispatch::IntegrationTest
```
Comment (ToniRib) [inline test/integration/user_can_view_items_from_an_individual_artist_test.rb]: Also planning to ignore this hound issue, I don't think having a long test title is a problem.

**Comment by ToniRib** [comment]
Merged in the newest master. All tests are passing.

**Pull request merged**